build1.sh

```diff
@@ -38,6 +38,9 @@ chmod +x meilix-generator.sh
 echo "$(<meilix-generator.sh)" #to test the file
 ./meilix-generator.sh
 rm meilix-generator.sh
+
+#change host name
```

Comment:
@zeon-zolt As this whole PR is just two lines of code, does this mean you retract the two lines of code and seek a different solution?